packages/bundlers/default/src/DefaultBundler.js

```diff
@@ -277,7 +278,8 @@ function decorateLegacyGraph(
         );
         for (let incomingDep of incomingDeps) {
           if (
-            incomingDep.priority === 'lazy' &&
+            (incomingDep.priority === 'lazy' ||
```

Comment:
This is treated as "lazy" for now. I think `tier` will need some special treatment as we need to act like a 'parallel' priority asset in most cases.

Reply:
This is updated now to use the packager to make the imports sync